manifests/plugin/powerdns.pp

```diff
@@ -0,0 +1,56 @@
+# See http://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_powerdns
+class collectd::plugin::powerdns (
+  $ensure                                 = 'present',
+  $order                                  = 10,
```

Comment:
please add datatypes for all parameters.

Reply:
done